src/mcts/types.h

```diff
@@ -12,7 +12,6 @@
 namespace mcts {
 
 using Action = int64_t;
-using NodeId = int64_t;
```

Comment:
not used anywhere

Reply:
So, you are removing it from node.h as well?